app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/Email.scala

```diff
@@ -34,17 +33,12 @@ object Email {
 
   implicit val eq: Eq[Email] = Eq.instance(_.value === _.value)
 
-  val form: Form[Email] = {
+  val mapping: Mapping[Email] = {
     val emailRegex = "^(?=.{1,132}.$).+@.+".r.pattern.asPredicate()
```

Comment:
the regex here should be `^(?=.{3,132}$)[^@]+@[^@]+$` to be more precise

Reply:
done